runtime/j9vm/jvm.c

```diff
@@ -3184,9 +3176,9 @@ deleteDirsFromLibpath(const char *const libpath, const char *const deleteStart,
 
 	return newPath;
 }
-#endif /* AIXPPC */
+#endif /* defined(AIXPPC) || (defined(J9ZOS390) && (JAVA_SPEC_VERSION >= 21)) */
 
-#if defined(AIXPPC) && defined(DEBUG_TEST)
+#if (defined(AIXPPC) || (defined(J9ZOS390) && (JAVA_SPEC_VERSION >= 21))) && defined(DEBUG_TEST)
```

Comment:
Please correct the comment in the matching `#endif`.

Reply:
Fixed.